libs/sdk-backend-spi/src/common/settings.ts

```diff
@@ -57,6 +57,11 @@ export enum SettingCatalog {
      * Indicates whether "Treat null as zero" option should be displayed in measure value filter configuration and its default state"
      */
     ADMeasureValueFilterNullAsZeroOption = "ADMeasureValueFilterNullAsZeroOption",
+
+    /**
+     * Indicates whether data point visibility configuration is available in AD
+     */
+    enableHidingOfDataPoints = "enableHidingOfDataPoints",
```

Comment:
all good. one little addition: please turn this option on by default for tiger. tiger's settings are at the moment hardcoded in the SDK (tiger does not have settings API yet). check out `sdk-backend-tiger`: `src/backend/workspace/settings` and `src/backenduser/settings.ts`

just add the key-value pair in there.